packages/contracts-bedrock/src/cannon/MIPS2.sol

```diff
@@ -555,6 +555,8 @@ contract MIPS2 is ISemver {
                 // ignored
             } else if (syscall_no == sys.SYS_PREAD64) {
                 // ignored
+            } else if (syscall_no == sys.SYS_STAT) {
+                // ignored
```

Comment:
Just a clarification - in the description you said we were adding this noop to both VMs, but isn't the single-threaded VM already implicitly noop-ing all of the syscalls that aren't explicitly handled?

Reply:
You're correct. This change affects only mt cannon.